utils/cliutils/commandsflags.go

```diff
@@ -877,6 +879,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  repoDeploySnapshots,
 		Usage: "[Optional] Deployment repository for snapshot artifacts.` `",
 	},
+	includePatterns: cli.StringFlag{
+		Name:  includePatterns,
+		Usage: "[Optional] Filter deployed artifacts by setting a wildcard pattern that specifies which artifacts to INCLUDE. You may provide multiple patterns separated by ';'.` `",
+	},
+	excludePatterns: cli.StringFlag{
+		Name:  excludePatterns,
+		Usage: "[Optional] Filter deployed artifacts by setting a wildcard pattern that specifies which artifacts to EXCLUDE. You may provide multiple patterns separated by ';'.` `",
```

Comment:
to EXCLUDE --> to exclude
(let's change the separator following my comment in the jfrog-cli-core PR if needed)